premailer/premailer.py

```diff
@@ -420,6 +442,14 @@ def transform(self, pretty_print=True, **kwargs):
                 parent = item.getparent()
                 del parent.attrib['class']
 
+        # Capitalize Margin properties
+        # To fix weird outlook bug
+        # https://www.emailonacid.com/blog/article/email-development/outlook.com-does-support-margins
+        if self.capitalize_float_margin:
+            for item in page.xpath('//@style'):
```

Comment:
Is there no better place to do this? What about the code that sets the style attributes in the first place. 
Now, somewhere in premailer it reads the `<style>` and `<link>` and sets the `style` attribute on the DOM and then when it's done that you extract them all out again and re-write them.